BLSQ/iaso
# POLIO-651: improve email layout

The layout of html emaisl sent by polio budget tool were a bit plain

## Self proof reading checklist

## Changes

- Change email title
- update template to add dividers and some layout (see printscreen)


## How to test

[Configure email](https://wiki.bluesquare.org/en/Dev-team/product-management/iaso/email), then add a step to any budget event;

Go to SMTP Bucket, check out the email, see what it looks like

## Print screen / video

![Screenshot 2022-12-13 at 17 13 23](https://user-images.githubusercontent.com/38907762/207386279-6398ad02-f6e0-49f1-b91a-b9217fc9a29e.png)
